cfme/tests/cloud_infra_common/test_vm_instance_analysis.py

```diff
@@ -4,15 +4,17 @@
 
 from cfme import test_requirements
 from cfme.cloud.provider import CloudProvider, CloudInfraProvider
+from cfme.cloud.provider.ec2 import EC2Provider
```

Comment:
For now, ssa for EC2Provider cannot be run, as it required different setup

Reply:
@rlbabyuk Honestly, I don't know anything about EC2. But there is a bug for EC2 SSA: https://bugzilla.redhat.com/show_bug.cgi?id=1533590
So I thought I might as well wrap the test case. But if you want, it's of course no problem for me to remove the decorator.

Reply:
@jan-zmeskal hm, maybe better to leave it, to be aware of this bz for EC2